hypothesis-python/tests/conjecture/test_inquisitor.py

```diff
@@ -23,29 +23,15 @@ def _new():
             with pytest.raises(error) as err:
                 settings(print_blob=False, derandomize=True, **kw)(f)()
             got = "\n".join(err.value.__notes__).strip() + "\n"
-            assert any(got == s.strip() + "\n" for s in expected)
+            assert any(got == s.strip() + "\n" for s in expected), got
 
         return _new
 
     return _inner
 
 
-# this should have a marked as freely varying, but
-# false negatives in our inquisitor code skip over it sometimes, depending on the
-# seen_passed_buffers. yet another thing that should be improved by moving to the ir.
 @fails_with_output(
-    [
```

Comment:
🎉 

I think this means we can also remove the `list` case from `fails_with_output`?